### ISSUE TITLE
Update Prisma version to v5

### ISSUE THREAD

**Issue by PCRinus** [keonik/prisma-erd-generator#220]
Hello folks! Prisma has just launched a new [major release](https://github.com/prisma/prisma/releases/tag/5.0.0), and it would be nice if this would support it as well

**Comment by keonik** [comment]
I haven't updated to Prisma v5 yet on my projects. Has anyone experienced issues with this library on it? 

From the looks of it, the breaking changes were mainly around querying and less on the generators side of things

**Comment by DRoet** [comment]
Upgrading to Prisma v5 and v1.9.1-0 for the erd-generator breaks for me. it is unable to run the mmdc binary together with the puppeteer config. Generating an `.md` output does seem to work.

Unfortunately I can't share my specific schema and debug information.

**Comment by keonik** [comment]
@DRoet I just did a catch up the `alpha` release on a Prisma v5 project and it went smooth. Try it again when you get time 🤞🏼

**Comment by keonik** [comment]
try `npm i -D prisma-erd-generator@alpha`

**Comment by keonik** [comment]
It should fetch [1.10.0-alpha.1](https://www.npmjs.com/package/prisma-erd-generator/v/1.10.0-alpha.1)

**Comment by akd-io** [comment]
1.10.0-alpha.1 gives me:
```
$ pnpm migrate dev                    

> prototype3@0.1.0 migrate /Users/akd/workspace/prototype
> prisma migrate dev "dev"

Environment variables loaded from .env
Prisma schema loaded from prisma/schema.prisma
Datasource "db": PostgreSQL database "postgres", schema "public" at "localhost:5432"

Already in sync, no schema change or pending migration was found.

Running generate... - Entity-relationship-diagram

Prisma ERD Generator: Unable to find chromium path for you MacOS arm64 machine. Attempting to use the default at /usr/bin/chromium-browser. To learn more visit https://github.com/keonik/prisma-erd-generator#-arm64-users-
✔ Generated Prisma Client (5.1.1 | library) to ./node_modules/.pnpm/@prisma+client@5.1.1_prisma@5.1.1/node_modules/@prisma/client in 74ms
Command failed: "/Users/akd/workspace/prototype/node_modules/.bin/mmdc" -i "/var/folders/nw/x__8gzpx3zg03m2683pk7j3m0000gn/T/prisma-erd-anto4u/prisma.mmd" -o "/Users/akd/workspace/prototyp
e/prisma/ERD.svg" -t forest -c "/var/folders/nw/x__8gzpx3zg03m2683pk7j3m0000gn/T/prisma-erd-anto4u/config.json" -p "/var/folders/nw/x__8gzpx3zg03m2683pk7j3m0000gn/T/prisma-erd-anto4u/puppe
teerConfig.json"
```

**Comment by keonik** [comment]
Sweet. Progress. Have you done the arm64 check stuff found [here](https://github.com/keonik/prisma-erd-generator#-arm64-users-). I'm on an M1 mac and got passed that on v5 today so I'm hopeful

**Comment by akd-io** [comment]
Aah, I didn't realize I needed to install chromium until now. I have not done anything with respect to the arm64 issues.

It seems weird though that 1.10.0 still generates ERD.svg but 1.10.0-alpha.1 does not.

**Comment by keonik** [comment]
yeah the versions are mismatched on alpha vs main. Alpha should be 1.11.0-alpha but I haven't been able to get this thing to self upgrade properly and have to do some manual uploads to npm every now and then

**Comment by keonik** [comment]
With arm64 it may work out of the box for the m1's but for some it does not. The message

```
Prisma ERD Generator: Unable to find chromium path for you MacOS arm64 machine. Attempting to use the default at /usr/bin/chromium-browser. To learn more visit https://github.com/keonik/prisma-erd-generator#-arm64-users-
```

is for those that get thrown errors because the assumed default path for chromium isn't working. Its weirdly been about 50% of the folks at my work had to manually set their chromium browser path

**Comment by akd-io** [comment]
Neither 1.10.0 nor 1.10.0-alpha.1 can find chromium on my machine. I still haven't installed it, so that's fine. I might just be one of those 50% :)

But how can 1.10.0 still generate the SVG when 1.10.0-alpha.1 cannot?

**Comment by keonik** [comment]
Top level you have this library that relies on mermaid-cli. mermaid-cli relies on puppeteer. So every version change between those libraries down the pipe could change the path its searching for chromium. There were version changes with that library between those two versions so my best guess is that assuming the prisma schemas are outputting the same, the versions of mermaid cli updated to default a different path to chromium based on OS

**Comment by keonik** [comment]
No worries mate! Glad its working. I had success as well! Gonna get this released to latest release soon! 🥳

**Comment by akd-io** [comment]
Thanks for the insanely quick response times. You didn't have to do that

**Comment by keonik** [comment]
Released on [v1.11.0](https://www.npmjs.com/package/prisma-erd-generator/v/1.11.0)

**Comment by DRoet** [comment]
I'm on windows and unfortunately it is not working for me. I did notice that the SVG gets generated if I manually run the mmdc command without the -p PUPPETEERCONFIG flag.

There is a difference in \Temp\PuppeteerConfig.json for me:
v1.8.0:
```
{"logLevel":"warn"}
```

v1.11.0:
```
{"logLevel":"warn","executablePath":"/usr/bin/chromium-browser","args":["--no-sandbox"]}
```

My guess is that the executablePath is incorrect. So it seems related to https://github.com/keonik/prisma-erd-generator/commit/6a619649bd47f46b0e4583bcb670f1756cdeec0c and not the v5 upgrade directly.
